prometheus/CMakeLists.txt

```diff
@@ -3,13 +3,19 @@ cmake_minimum_required(VERSION 2.8)
 project("prometheus-client")
 set(PROJECT_VERSION "0.99")
 set(PROJECT_DESCRIPTION "Prometheus C++ client library")
+set(CMAKE_MACOSX_RPATH true)
 
 include(GNUInstallDirs)
 include(FindPkgConfig)
+if (APPLE)
+  # Find homebrew's libicu on Mac OS although it is hidden pretty well
+  set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:/usr/local/opt/icu4c/lib/pkgconfig")
```

Comment:
Is this really how the best-practice is for homebrew? Seems very hacky and dependent on homebrew internals.